### DISCUSSION TITLE
equipo hackaton y reuniones de estudio

### DISCUSSION THREAD

**Discussion by MacaCoronel93** [Launch-X-Latam/MisionFrontEnd#496] [Team Posts]
Hola estamos buscando 3  personas para sumarse a nuestro grupo y para estudiar el contenido ! 
Dejen sus nombres y correos de inovaccion

**Comment by Annamalag** [comment]
Yo quiero

**Comment by Annamalag** [comment]
Anna ***@***.***

**Comment by VAlonzoGO** [comment]
Yo me uno lx05204@innovaccion,mx

**Comment by IsraelVera** [comment]
Israel vera
***@***.***

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by LuisAlbertoCP** [comment]
Hola, Si aun estas buscando un integrante para tu equipo aquí estoy yo 
Te dejo mi usuario 
lx03646@innovaccion.mx

**Comment by Francolin09** [comment]
Hola me gustaría ser parte de su grupo de estudios mi correo es lx08668@[innovaccion.mx]

**Comment by Kariid26** [comment]
Hola, yo ya tengo equipo para el hackaton pero me gustaría formar parte del grupo de estudio si fuera posible, mi usuario es:
lx05360

Obtener Outlook para iOS<https://aka.ms/o0ukef>
________________________________
De: Francolin09 ***@***.***>
Enviado: Tuesday, November 8, 2022 10:25:44 AM
CC: Subscribed ***@***.***>
Asunto: Re: [Launch-X-Latam/mision-backend] equipo hackaton y reuniones de estudio (#25)


Hola me gustaría ser parte de su grupo de estudios mi correo es ***@***.***

**Comment by gustavo-galindo** [comment]
Hola  todos mi correo es lx13084@innovacion.mx
Gustavo-galindo es mi github

**Comment by FrancesscoPolanco** [comment]
Hola a todos me gustaría estar en el grupo de estudio este es mi correo ***@***.*** , FrancesscoPolanco en GitHub gracias.

Obtener Outlook para Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by DeusGarrix4k** [comment]
Hola me gustaria unirme con uds recien estoy empezando a ponerme al dia espero podamos trabajar juntos dejo mi correo:lx15821@innovaccion.mx

**Comment by May-Alvarez** [comment]
Hola me gustaria unirme recien estoy empezando a ponerme al dia con las practicas dejo mi correo: lx00319@innovaccion.mx

**Comment by Alisvaz** [comment]
Hola a todos, ya se llenaron tus cupos para el equipo? estoy en busqueda de equipo para Hackaton.
de igual forma, para los que aún no tienen equipo, podemos formar uno!!
mi usuario de launch = lx04406
mi GitHub =  solo hacer click en el nombre de este comentario :)

Saludos!

**Comment by CristopherSandoval** [comment]
@Alisvaz Hola Alan! Yo quisiera formar equipo contigo! :)
Usuario launcx =  lx15324
Mi github = @CristopherSandoval

**Comment by alunator** [comment]
Hola a todos! Busco un equipo al cual integrarme para el Hackathon ^^
Mi correo es: lx15065@innovaccion.mx 
Saludos~

**Comment by AleBT75** [comment]
Buen día a todos los participantes, si aún está en las posibilidades deseo unirme a un grupo, mi correo es: lx12185@innovaccion.mx
Deseo aportar lo mejor de mi para esta experiencia, Saludos

**Comment by MacaCoronel93** [comment]
Les envie un correo !! a los primeros comentaron
